bom/pom.xml

```diff
@@ -15,84 +15,102 @@
 
   <name>Google Guice - Bill of Materials</name>
 
+  <properties>
+    <guice.version>${project.version}</guice.version>
+    <guice-assistedinject.version>${project.version}</guice-assistedinject.version>
+    <guice-dagger-adapter.versionn>${project.version}</guice-dagger-adapter.versionn>
+    <guice-dagger-adapter.version>${project.version}</guice-dagger-adapter.version>
+    <guice-grapher.version>${project.version}</guice-grapher.version>
+    <guice-jmx.version>${project.version}</guice-jmx.version>
+    <guice-jndi.version>${project.version}</guice-jndi.version>
+    <guice-multibindings.version>${project.version}</guice-multibindings.version>
+    <guice-persist.version>${project.version}</guice-persist.version>
+    <guice-servlet.version>${project.version}</guice-servlet.version>
+    <guice-spring.version>${project.version}</guice-spring.version>
+    <guice-struts2.version>${project.version}</guice-struts2.version>
+    <guice-testlib.version>${project.version}</guice-testlib.version>
+    <guice-throwingproviders.version>${project.version}</guice-throwingproviders.version>
+    <guava.version>18.0</guava.version>
+  </properties>
+
```

Comment:
Removed the hardcoded versions as they didn't update during `mvn release:update-versions` but kept the properties. I believe this gets the best of both worlds.

Thoughts?